### PR TITLE
Release notes 1.4.0

### PR DESCRIPTION
**Pull request by robertpatrick**

### CODE REVIEW

File: documentation/staging/1.4/content/release/_index.md [hunk 1]
```diff
@@ -5,34 +5,29 @@ weight = 7
...
 #### Other Changes
-- Updated the [WebLogic Remote Console](https://github.com/oracle/weblogic-remote-console) integration to support version 2.4.0.
-- Added support for specifying node selectors when installing the WebLogic Kubernetes Operator and deploying the WebLogic
-  domain, either directly or as a Verrazzano component.
-- Added support for the user specifying the Helm timeout value when helm is used (i.e., Installing/updating WebLogic
-  Kubernetes Operator and installing the Ingress Controller).
-- Added tooltips across all icon buttons to describe what they do.
-- Added an anti-affinity specification to the generated WebLogic domain resource, as recommended by the WebLogic Kubernetes Operator docs.
+- #175: Added Zoom In, Zoom Out, and Actual Size menu options to the Window menu.
+- #178: Integrating the Prepare Model action with the changes in WebLogic Deploy Tooling 2.4.0.
```
Comment (rosemarymarano) [inline documentation/staging/1.4/content/release/_index.md:20]: Integrating -> Integrated (? to be parallel with other bulleted items)

(To consider, in Known Issues, we use code font for the `Check Application Status` button. If you like that, then we should use code font for `Prepare Model`, `Install Ingress Controller`, and `Update Ingress Routes`. If not, then let's not use it on the buttons/actions in the RNs.)

File: documentation/staging/1.4/content/release/_index.md [hunk 2]
```diff
@@ -5,34 +5,29 @@ weight = 7
...
+- #179: Updated the [WebLogic Remote Console](https://github.com/oracle/weblogic-remote-console) integration to support version 2.4.1 (2.4.0 also supported).
 
 #### Bug Fixes
-- Fixed an issue introduced by newer WDT versions that was forcing the domain cluster size to only allow replicas to be set to 0 (WKTUI-365).
-- #164: Fixed the link in the Introduction dialog to open in the user's browser instead of inside the app.
-- #167: Improved the validation logic and error messages around having an empty model.
-- #170: Worked around an issue that was limiting the ability to select text from the Code View scripts (WKTUI-375).
+- #171: Improved workaround for the issue limiting the ability to select text from the Code View scripts (WKTUI-375).
+- #172: Resolved issue when using Domain-in-Image that was causing the domain to not be created.
+- #174: Resolved an issue where the Install Ingress Controller action was always passing the image pull secret--even when the user said not to use one.
```
Comment (rosemarymarano) [inline documentation/staging/1.4/content/release/_index.md:26]: the user said -> the user indicated

File: documentation/staging/1.4/content/release/_index.md [hunk 3]
```diff
@@ -5,34 +5,29 @@ weight = 7
...
 
 #### Bug Fixes
-- Fixed an issue introduced by newer WDT versions that was forcing the domain cluster size to only allow replicas to be set to 0 (WKTUI-365).
-- #164: Fixed the link in the Introduction dialog to open in the user's browser instead of inside the app.
-- #167: Improved the validation logic and error messages around having an empty model.
-- #170: Worked around an issue that was limiting the ability to select text from the Code View scripts (WKTUI-375).
+- #171: Improved workaround for the issue limiting the ability to select text from the Code View scripts (WKTUI-375).
+- #172: Resolved issue when using Domain-in-Image that was causing the domain to not be created.
+- #174: Resolved an issue where the Install Ingress Controller action was always passing the image pull secret--even when the user said not to use one. 
+- #176: Resolved an issue where the Update Ingress Routes action was always passing the SSL pass-through argument--even when the user did not request it.
```
Comment (rosemarymarano) [inline documentation/staging/1.4/content/release/_index.md:27]: SSL pass-through argument -> SSL `pass-through` argument (code font)

**Review by rosemarymarano — APPROVED** [review]
A few minor comments

**Review by rosemarymarano — APPROVED** [review]

**Review by rakillen — APPROVED** [review]

File: documentation/staging/1.4/content/release/_index.md [hunk 4]
```diff
@@ -27,9 +27,17 @@ pre = "<b> </b>"
 - #176: Resolved an issue where the `Update Ingress Routes` action was always passing the `SSL pass-through` argument--even when the user did not request it.
 - #182: Resolved Verrazzano 1.4.0+ installation issue caused by a change in the published operator file name. 
 - #186: Resolved Verrazzano-related issue where Ingress Trait rules were not including the hosts or destination-related fields.
+- #187: Resolved an issue where the operator domain variable override config map was not being created properly.
```
Comment (rosemarymarano) [inline documentation/staging/1.4/content/release/_index.md:30]: config map -> ConfigMap

File: documentation/staging/1.4/content/release/_index.md [hunk 5]
```diff
@@ -27,9 +27,17 @@ pre = "<b> </b>"
...
+- #187: Resolved an issue with the `apiVersion` of the Verrazzano component- and application-related being incorrect.
```
Comment (rosemarymarano) [inline documentation/staging/1.4/content/release/_index.md:31]: This seems like it's missing a word, "the Verrazzano component- and application-related (what) being incorrect.

File: documentation/staging/1.4/content/release/_index.md [hunk 6]
```diff
@@ -27,9 +27,17 @@ pre = "<b> </b>"
...
 
 #### Known Issues
 - On Linux, the open and save dialogs open behind the main application window.  This is due to
   [Electron bug 32857](https://github.com/electron/electron/issues/32857).
 - When deploying a Verrazzano application, the `Check Application Status` button only checks the status of the project's
   Verrazzano component containing the WebLogic domain specification.
+- WKTUI does not currently show the Verrazzano URL needed to access the application.  The URL will always be of
+  the form `https://<gateway-hostname>/<app-path>`, where the `gateway-hostname` is the name created by Verrazzano
+  in the `Gateway` object.  To get the name of the generated gateway object, run `kubectl get gateway -n <namespace> --template
```
Comment (rosemarymarano) [inline documentation/staging/1.4/content/release/_index.md:40]: generated gateway object -> generated `Gateway` object

**Review by rosemarymarano — COMMENTED** [review]
Few comments

**Pull request merged**